samples/Server/src/routes/createRoom.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as express from 'express';
+import { getResourceConnectionString } from '../lib/envHelper';
+import { RoomsClient, CreateRoomOptions } from '@azure/communication-rooms';
+import { createUser } from '../lib/identityClient';
+
+const router = express.Router();
+
+const TWENTY_FOUR_HOURS = 24 * 60 * 60 * 1000;
+/**
+ * route: /createRoom/
+ *
+ * purpose: Create a new room.
+ *
+ * @returns The new roomId as string
+ */
+
+router.post('/', async function (req, res, next) {
+  // create RoomsClient
+  const roomsClient: RoomsClient = new RoomsClient(getResourceConnectionString());
+
+  const validFrom = new Date();
+  const validUntil = new Date(validFrom.getTime() + TWENTY_FOUR_HOURS);
+
+  // Options payload to create a room
+  const createRoomOptions: CreateRoomOptions = {
+    validFrom: validFrom,
+    validUntil: validUntil
+  };
+
+  // create a room with the request payload
+  const createRoom = await roomsClient.createRoom(createRoomOptions);
```

Comment:
nit: variable name here is slightly off, createRoom would imply a verb and that this creates a room, instead consider `createdRoom`